tests/subsys/usb/device/src/main.c

```diff
@@ -190,24 +190,16 @@ static void test_usb_dc_api_read_write(void)
 }
 
 /* test case main entry */
-void test_main(void)
+static void *device_usb_setup(void)
 {
 	int ret;
 
 	ret = usb_enable(NULL);
 	if (ret != 0) {
 		printk("Failed to enable USB\n");
-		return;
+		return NULL;
```

Comment:
`return NULL;` here seems to be redundant.

Reply:
Ok, I know. But if I don't add the NULL, it will appear this error, " ‘return’ with no value". So I need to add it after return.

Reply:
you can replace this by zassue_true(ret == 0, ...)

Reply:
Ok, I will modify it.